server/routes/index.js

```diff
@@ -0,0 +1,24 @@
+import client from '../helpers/redis';
+
+import daysBetween from '../helpers/daysBetween';
+
+export default (app) => {
+  app.get('/worker-status', (req, res) => client.mget('startTime', 'message', 'jobDone', (err, response) => {
+    if (err) {
+      return res.status(400).send({
+        message: 'Error with redis connection, kindly connect to redis',
+      });
+    }
+
+    const startTime = new Date(JSON.parse(response[0]));
+    const message = response[1];
+    const jobDone = response[2];
+
+    return res.status(200).send({
+      message,
+      jobDone, // no of times the worker executed,
```

Comment:
What if this variable `jobDone` is renamed to something like `numberOfJobs`. I never knew it meant: `no of times the worker executed` till this point

Reply:
cool suggestion